themes/classic/templates/errors/404.tpl

```diff
@@ -1,12 +1,13 @@
 {extends file='page.tpl'}
 
 {block name='page_title'}
-  {l s='This page is not available'}
+  {l s='The page you are looking for was not found.'}
```

Comment:
![This is not the page you are looking for](http://i2.kym-cdn.com/photos/images/newsfeed/001/005/935/ef1.jpg)